perfkitbenchmarker/providers/gcp/gce_disk.py

```diff
@@ -67,34 +66,27 @@ def __init__(self, disk_spec, name, zone, project, image=None):
 
   def _Create(self):
     """Creates the disk."""
-    create_cmd = [FLAGS.gcloud_path,
-                  'compute',
-                  'disks',
-                  'create', self.name,
-                  '--size', str(self.disk_size),
-                  '--type', self.disk_type]
-    create_cmd.extend(util.GetDefaultGcloudFlags(self))
+    cmd = util.GcloudCommand('compute', 'disks', 'create', self.name)
+    cmd.flags['size'] = self.disk_size
+    cmd.flags['type'] = self.disk_type
+    cmd.AddCommonFlags(self)
     if self.image:
-      create_cmd.extend(['--image', self.image])
+      cmd.flags['image'] = self.image
       if FLAGS.image_project:
-        create_cmd.extend(['--image-project', FLAGS.image_project])
-    vm_util.IssueCommand(create_cmd)
+        cmd.flags['image-project'] = FLAGS.image_project
+    cmd.Issue()
 
   def _Delete(self):
     """Deletes the disk."""
-    delete_cmd = [FLAGS.gcloud_path,
-                  'compute', 'disks',
-                  'delete', self.name]
-    delete_cmd.extend(util.GetDefaultGcloudFlags(self))
-    vm_util.IssueCommand(delete_cmd)
+    cmd = util.GcloudCommand('compute', 'disks', 'delete', self.name)
+    cmd.AddCommonFlags(self)
+    cmd.Issue()
 
   def _Exists(self):
     """Returns true if the disk exists."""
-    getdisk_cmd = [FLAGS.gcloud_path,
-                   'compute', 'disks',
-                   'describe', self.name]
-    getdisk_cmd.extend(util.GetDefaultGcloudFlags(self))
-    stdout, _, _ = vm_util.IssueCommand(getdisk_cmd, suppress_warning=True)
+    cmd = util.GcloudCommand('compute', 'disks', 'describe', self.name)
+    cmd.AddCommonFlags(self)
```

Comment:
Are there any cases where this isn't called? If not, could the Command object take the object being created/described/deleted in the constructor?